tests/typings/copy.js

```diff
@@ -44,7 +45,12 @@ async function copy() {
             const destination = path.join(__dirname, outDir, 'node_modules', packageName)
             const packageDir = packages[packageName]
 
-            const destDir = destination.split('/').slice(0, -1).join('/')
+            let destDir
+            if (process.platform === 'win32') {
+                destDir = destination.split('\\').slice(0, -1).join('\\')
+            } else {
+                destDir = destination.split('/').slice(0, -1).join('/')
+            }
```

Comment:
How about?
```suggestion
            const destDir = destination.split(path.sep).slice(0, -1).join(path.sep)
```

Reply:
that's awesome - i didn't know that one...